### DISCUSSION TITLE
HSRB wrist force sensor issues

### DISCUSSION THREAD

**Discussion by Janillaa** [HumanSupportRobot/QandA#55] [Q&A]
Hi there,

We are trying to use the force sensor in the wrist for our program however we have run into a few problems which are listed below:

1. The data published doesn't change corresponding to a force applied to the gripper/wrist. We have completed the hand self weight calibration.
2. The wrist sensor occasionally stops publishing. 
3. The HSR diagnostics page occasionally says that the wrist wrench is disconnected.

Any tips for how to fix this?

Thank you in advance!

**Comment by ayuguchi** [comment]
It seems to be related to the following discussion. What about you?
https://github.com/orgs/HumanSupportRobot/discussions/41
